application/app.py

```diff
@@ -319,7 +319,7 @@ def open_image_fd(self, dialog_text="Open Image", default_folder=""):
         """
         fname = QtWidgets.QFileDialog.getOpenFileName(self, dialog_text, default_folder)  # TODO: Filter to show only image files
         if fname:
-            image = QtGui.QImage(fname)
+            image = QtGui.QImage(fname[0])
```

Comment:
In Qt5 fname would be a tuple of both the file location and possible types. We only want the file location